### ISSUE TITLE
Pip install does not work anymore

### ISSUE THREAD

**Issue by Bachibouzouk** [openego/ego.io#78]
`pip install egoio` generate the following issue 

```
    error in egoio setup command: "values of 'package_data' dict" must be a list of strings (got 'sqlacodegen_oedb.sh')
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

It is fixed in the `dev` branch but not on master and it that would be useful to close this issue : https://github.com/OpenEnergyPlatform/oeplatform/pull/417

**Comment by Bachibouzouk** [comment]
@nesnoj , @gplssm - if you give me the rights I can make the release

**Comment by nesnoj** [comment]
> @nesnoj , @gplssm - if you give me the rights I can make the release

Thanks for your offer, but we will do the release as soon as the password-less access is done, @gplssm will do this tonight (cf. #70).
Can you wait until tomorrow?

**Comment by Bachibouzouk** [comment]
Yes :)

**Comment by nesnoj** [comment]
Fixed in [release v0.4.6](https://github.com/openego/ego.io/releases/tag/v0.4.6) (also available on PYPI)
